hoot-core-test/src/test/cpp/hoot/core/conflate/point/PlacesPoiMatchTest.cpp

```diff
@@ -162,7 +162,7 @@ class PlacesPoiMatchTest : public CppUnit::TestFixture
         shared_ptr<const Element> e1 = map->getElement(it->first);
         shared_ptr<const Element> e2 = map->getElement(it->second);
         //LOG_INFO(e1->getTags()["note"] << " <=> " << e2->getTags()["note"]);
-        CPPUNIT_ASSERT_EQUAL(e1->getTags()["note"], e2->getTags()["note"]);
+        CPPUNIT_ASSERT_EQUAL(e1->getTags()["note"].toStdString(), e2->getTags()["note"].toStdString());
```

Comment:
Yes? No? This is the one change that I don't really feel great about. GCC kept barfing up an error here, about an ambiguous << operator... it couldn't decide between the QString or std string. Explicitly using .toStdString() made the problem go away. The HOOT_STR_EQUALS macro looks like it uses a stringstream to perform << on the operands, then checks the equality of the std strings that are built. I strongly suspect the ambiguous operator would still have been a problem!